installer/templates/new/web/gettext.ex

```diff
@@ -0,0 +1,4 @@
+defmodule <%= application_module %>.Gettext do
+  use Gettext, otp_app: :<%= application_name %>
```

Comment:
Let's add some docs here and include a link to the Gettext docs too.

Also, let's change the docs for the `Phoenix` module so it links to the `gettext` project too.